eslint.config.mjs

```diff
@@ -224,7 +222,7 @@ export default tseslint.config(
         },
     },
     {
-        files: ['**/__tests__/**', '**/__mocks__/**', 'packages/@lwc/integration-karma/**'],
+        files: ['packages/@lwc/integration-karma/**'],
```

Comment:
```suggestion
        files: ['**/__tests__/**', 'packages/@lwc/integration-karma/**'],
```
Of the rules listed, only `valid-expect` is in `recommended`. If we want them to apply to `__tests__`, we'll still need to include it here.

Reply:
Good catch. I added the same rules.

We might want to revisit `'vitest/no-done-callback'` later since it's deprecated and will give a false positive when using test context like this:

```typescript
test.concurrent('foo', async ({ expect }) => {
   await expect('hello').toMatchSnapshot()
})
```

But no need to worry for now. I'll see if I can push a fix to `@vitest/eslint-plugin`.